database/clickhouse/clickhouse_test.go

```diff
@@ -26,7 +26,7 @@ var (
 		PortRequired: true, ReadyFunc: isReady,
 	}
 	specs = []dktesting.ContainerSpec{
-		{ImageName: "yandex/clickhouse-server:21.3", Options: opts},
```

Comment:
Confirming that v21 is no longer supported:
https://hub.docker.com/_/clickhouse